lib/OpenQA/Task/Job/HookScript.pm

```diff
@@ -13,16 +13,17 @@ sub _hook_script ($job, $hook, $openqa_job_id, $options) {
     my $ensure_task_retry_on_termination_signal_guard = OpenQA::Task::SignalGuard->new($job);
     my $timeout = $options->{timeout};
     my $kill_timeout = $options->{kill_timeout};
-    my $delay = $options->{delay};
     my $retries = $options->{retries};
+    my $delay = $options->{delay} * ($job->retries + 1);
     my $skip_rc = $options->{skip_rc};
 
     $ensure_task_retry_on_termination_signal_guard->abort(1);
     my ($rc, $out) = _run_hook($hook, $openqa_job_id, $timeout, $kill_timeout);
     $job->note(hook_cmd => $hook, hook_result => $out, hook_rc => $rc);
 
-    if ($retries && $skip_rc) {
-        $job->retry($delay ? {delay => $delay} : {}) if defined $rc && $rc == $skip_rc && $job->retries < $retries;
+    if ($retries && $skip_rc && defined $rc && $rc == $skip_rc && $job->retries < $retries) {
+        $job->app->log->debug("Job $job->id: Retrying $job->retries with linear delay: $delay");
```

Comment:
I assume that's seconds so

```suggestion
        $job->app->log->debug("Job $job->id: Retrying $job->retries with linear delay: ${delay}s");
```

Also can you give an example of a log line from actual execution? I don't think any of the three used variables are ensured to be defined

Reply:
I tried but gru doesnt work properly local. However all of them are expected to be available, and if I remember correctly I could print them and find them